templates/realtime/1/docker-compose.yml

```diff
@@ -408,9 +405,9 @@ services:
       INASAFE_REALTIME_BROKER_HOST: amqp://guest:guest@rabbitmq:5672/
       INASAFE_REALTIME_PROJECT: /home/realtime/analysis_data/realtime.qgs
       INASAFE_REALTIME_REST_LOGIN_URL: http://web:8080/realtime/api-auth/login/
-      INASAFE_REALTIME_REST_PASSWORD: 7ny3pQZ8he
```

Comment:
Password breach :p, hence the needed fix.
I regenerated passwords on live and staging